common/build/eslint-plugin-fluid/src/rules/no-unchecked-record-access.js

```diff
@@ -515,3 +578,111 @@ function isStrictlyTypedVariable(typeAnnotation) {
 	// Consider any non-union type as strictly typed, except for 'any' and 'unknown'
 	return typeAnnotation.type !== "TSAnyKeyword" && typeAnnotation.type !== "TSUnknownKeyword";
 }
+
+// Helper function to find the containing block (e.g., BlockStatement)
+function findContainingBlock(node) {
+	let current = node;
+	while (current) {
+		if (current.type === "BlockStatement" || current.type === "Program") {
+			return current;
+		}
+		current = current.parent;
+	}
+	return null;
```

Comment:
Is `null` strictly required here or can we use `undefined` instead?

Reply:
I've replaced the `null` returns with `undefined`, some of them error when I change it to undefined, I have changed this one and some other ones that are fine with being `undefined`